packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -4480,6 +4480,30 @@ describe('shallow', () => {
         expect(wrapper.context().name).to.equal(context.name);
         expect(wrapper.context('name')).to.equal(context.name);
       });
+
+      it('filters context to childContextTypes', () => {
+        class Bar extends React.Component {
+          render() {
+            return <div>{JSON.stringify(this.context)}</div>;
+          }
+        }
+        Bar.contextTypes = {
+          name: PropTypes.string,
+        };
+        class Foo extends React.Component {
+          render() {
+            return (
+              <div>
+                <Bar />
+              </div>
+            );
+          }
+        }
+
+        const context = { name: 'foo', hello: 'world' };
+        const wrapper = shallow(<Foo />, { context });
+        expect(JSON.parse(wrapper.find(Bar).dive().text())).to.eql({ name: 'foo' });
```

Comment:
instead of using json and text here, could the test assert that `wrapper.find(Bar).context()` was the expected object?